__tests__/integration.test.js

```diff
@@ -77,4 +77,36 @@ describe("/api/articles", () => {
         expect(body.message).toBe("Article not found");
       });
   });
+  test("GET 400 - Incorrect article id type responds with bad request error", () => {
+    return request(app)
+      .get("/api/articles/not_valid_type")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad request");
+      });
+  });
+} );
+
+describe("/api/articles", () => {
+  test("GET 200 - Responds with an array of all articles and the comment counts of each", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("created_at", { descending: true });
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)